src/routes/triviaRoutes.js

```diff
@@ -36,9 +31,77 @@ router.post('/', async (req, res) => {
         // Save the triviaGame instance
         await triviaGame.save();
 
-        res.status(200).json({ message: 'Trivia game created successfully' });
+        res.status(201).json({ message: `The trivia game (${triviaGame.id}) has been created successfully.` });
+    } catch (error) {
+        res.status(500).json({ error: `Failed to create trivia game: ${error.message}.` });
+    }
+});
+
+// GET /api/trivia/games?ownedGames=false&limit=20
+router.get('/games', async (req, res) => {
+    /*
+      Get all trivia games associated with the user
+    */
+    const { sub } = req.user;
+    const { ownedGames, limit } = req.query;
+
+    if (!isBooleanString(ownedGames) || !isValidInteger(limit)) {
+        return res.status(400).json({ error: 'Invalid query type parameters.' });
+    }
+
+    try {
+        const options = {
```

Comment:
Why is this a part of the try? Is it possible not to be a part of the try catch?

Reply:
Solved. https://github.com/Syncora/Syncora-Backend/pull/12/commits/dc1f9ebdcae53ff111286490b7f1232e4d8a4c30